app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os.path
+from os import makedirs, rename, remove
+
+
+def move_file(command: str) -> None:
+    if command.split()[0] == "mv" and len(command.split()) == 3:
+        command_name, source_file_name, destination_name = command.split()
+
+        if os.path.dirname(destination_name):
+            with open(source_file_name) as source:
+                source_text = source.read()
+
+            destination_path = "/".join(destination_name.split("/"))
+            destination_folders = "/".join(destination_name.split("/")[:-1])
```

Comment:
Avoid using platform-specific directory delimiters in the code, use built-in functions or methods for similar functionality